### ISSUE TITLE
Book 05: Sentence 059

### ISSUE THREAD

**Issue by ftyers** [ftyers/UD_Classical_Nahuatl-FloCo#16]
![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/1054104c-6092-4172-a010-63785b211fd4)

```
# sent_id = Book_05_-_The_Omens.txt:59
# text = Ic chicuei capitulo, vncan mjtoa: in tetzaujtl, injc motetzaujaia: in jquac qujttaia pinaujztli.
# text[norm] = Ic chicueyi capítulo, oncan mihtoa: in tetzahuitl, inic motetzahuiaya: in ihcuac quittaya pinahuiztli.
# text[orig] = Ic·chicuei·capitulo·,·vncan·mjtoa·:·intetzaujtl·,·injc·motetzaujaia·:·inj¶quac·qujttaia·pinaujztli·.
# text[osp] = Capitulo ocho, en que se tracta del mal aguero, que tomauan los naturales, desta nueua españa: quando encontrauan con una sauandija, o gusano que la llaman pinauiztli.
# references = A&D/172:1
1	Ic	ic	ADV	_	_	2	_	_	_
2	chicuei	chicueyi	NUM	_	_	3	_	_	_
3	capitulo	capítulo	NOUN	_	State=Abs	6	_	_	_
4	,	,	PUNCT	_	_	6	punct	_	_
5	vncan	oncan	ADV	_	_	6	_	_	_
6	mjtoa	ihtoa	VERB	_	Number[subj]=Plur,Sing|Person[subj]=3|Reflexive[obj]=Yes|Subcat=Tran|Tense=Pres|VerbForm=Fin	0	root	_	_
7	:	:	PUNCT	_	_	9	punct	_	_
8	in	in	DET|SCONJ	_	_	9	_	_	_
9	tetzaujtl	tetzahuitl	NOUN	_	State=Abs	6	_	_	_
10	,	,	PUNCT	_	_	12	punct	_	_
11	injc	inic	SCONJ	_	_	12	_	_	_
12	motetzaujaia	tetzahuia	VERB	_	_	9	_	_	_
13	:	:	PUNCT	_	_	16	punct	_	_
14	in	in	DET|SCONJ	_	_	16	_	_	_
15	jquac	ihcuac	SCONJ	_	_	16	_	_	_
16	qujttaia	itta	VERB	_	Aspect=Imp|Number[obj]=Sing|Number[subj]=Plur,Sing|Person[obj]=3|Person[subj]=3|Subcat=Tran|Tense=Past|VerbForm=Fin	12	_	__
17	pinaujztli	pinahuiztli	NOUN	_	State=Abs	16	_	_	_
18	.	.	PUNCT	_	_	6	punct	_	_

```

**Comment by ftyers** [comment]
![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/37e58778-890b-43a1-97c7-a6630a5cb437)

Updated according to decisions:
- *ihcuac*:  028 (#4)
- Ordinals: 001 (#14)

Remaining questions:
- Status of *in* with *tetzahuitl* (see also other issues ad nauseum)
- Status of *inic* and the clause it introduces, does it modify *tetzahuitl* as `acl` or *mihtoa* as `advcl`?

**Comment by Lguyogiro** [comment]
![imagen](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/7957321/52d3b58b-ca24-4a41-920d-4f2574e19754)

keeping with our treatment of _mihtoa: X_, I wonder if we can do this, and say basically "it is said there was a portent that was taken as a portent when they saw the pinahuiztli" ? I know it is a bit strange because there is no _catca_, but maybe in this case it is just present tense and so doesn't require the copula.

**Comment by ftyers** [comment]
This is very like #34, so following that structure we would get:

![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/6761cb65-4cce-44d6-b25e-8c7ad19841bf)

What do you think?

**Comment by Lguyogiro** [comment]
LGTM

**Comment by ftyers** [comment]
Merged in 29e0769
